python-sdk/src/forta_agent/alert_test.py

```diff
@@ -1,19 +1,28 @@
-from .utils import get_alerts;
+from .utils import get_alerts
+
 
 def test_get_alerts_returns_more_than_one_alert():
```

Comment:
Should we remove these test?

Reply:
we can leave commented for now. they need to be updated to not make real network calls since the API response was causing them to fail